app/policies/follow_policy.rb

```diff
@@ -1,15 +1,7 @@
 class FollowPolicy < ApplicationPolicy
-  def update?
-    false
-  end
-
   def create?
     record.follower == user
   end
+  alias_method :update?, :create?
   alias_method :destroy?, :create?
-
-  def visible_attributes(all)
-    is_owner? ? all : all - %i[hidden]
-  end
-  alias_method :editable_attributes, :visible_attributes
```

Comment:
So I just read my own code and it looks like `visible_attributes` is run with a `record` of the target row but `editable_attributes` is run with the class, so... I guess we can honestly just remove `editable_attributes` on Follow (the only person who can edit is owner, and they can edit anything about it idgaf), but we need to figure something out for GroupMember